requirements.txt

```diff
@@ -33,3 +33,9 @@ django-sendfile
 django-polymorphic
 networkx
 rdflib>=4.1.0
+celery[redis]
+django-celery
+scikit-learn
+python-redis
```

Comment:
==> default: Notice: /Stage[main]/Main/Neurovault::Main[nvault-install]/Python::Requirements[/tmp/temp_requirements.txt]/Exec[python_requirements/tmp/temp_requirements.txt]/returns: Downloading/unpacking python-redis (from -r /tmp/temp_requirements.txt (line 39))
	==> default: Notice: /Stage[main]/Main/Neurovault::Main[nvault-install]/Python::Requirements[/tmp/temp_requirements.txt]/Exec[python_requirements/tmp/temp_requirements.txt]/returns:   Could not find any downloads that satisfy the requirement python-redis (from -r /tmp/temp_requirements.txt (line 39))